Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk
```

Comment:
Both CodeFactor and Codacy flag a `FROM` line that doesn't specify a specific version of the base image as a warning. For the sake of making this really "infrastructure as code" (repeatable and versionable, i.e. in six months we could rebuild a byte-for-byte identical image for comparison-testing purposes), I agree with those QA tools.